core/src/StructureFactory.cpp

```diff
@@ -41,25 +42,42 @@ std::string structureNameFromFile(const std::string& filePath)
     return structureName;
 }
 
+#ifdef USE_MPI
+ModelState StructureFactory::stateFromFile(
+    const std::string& filePath, const std::string& partitionFile, ModelMetadata& metadata)
+#else
 ModelState StructureFactory::stateFromFile(const std::string& filePath)
+#endif
 {
     std::string structureName = structureNameFromFile(filePath);
     // TODO There must be a better way
     if (DevGrid::structureName == structureName) {
         Module::setImplementation<IStructure>("DevGrid");
         DevGrid gridIn;
         gridIn.setIO(new DevGridIO(gridIn));
+#ifdef USE_MPI
+        return gridIn.getModelState(filePath, partitionFile, metadata);
+#else
         return gridIn.getModelState(filePath);
+#endif
```

Comment:
DevGrid has been removed. Is there a need to rebase? Or will the merge/PR sort this all out?

Reply:
I think `DevGrid` here can be replaced by `RectGrid` and it should (hopefully) be fine.